.changeset/sweet-chairs-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-github': patch
+---
+
+Bump integration package
```

Comment:
And this message updated (explain to the reader what changed)

Reply:
Maybe I missed something: the package @backstage/plugin-catalog-backend-module-github shouldn't be deployed to bump @backstage/integration version?